src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RegistrationServiceImplTest {
+    private static final long ID = 1;
+    private static final int AGE = 18;
+    private static final String LOGIN = "test@gmail.com";
+    private static final String PASSWORD = "1234556";
+    private static RegistrationService registrationService;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        Storage.people.clear();
+        user.setId(ID);
+        user.setLogin(LOGIN);
+        user.setPassword(PASSWORD);
+        user.setAge(AGE);
+    }
+
+    @Test
+    void register_ok() {
```

Comment:
let's make the naming of test methods like it's described in the description of the task